io.asgardio.android.oidc.sdk.sample/src/main/res/raw/oidc_config.json

```diff
@@ -1,6 +1,6 @@
 {
-  "client_id": "tqVQ5YCvmlbdT9Z5FfK_bEnyUzca",
+  "client_id": "bdEQHU0CW8_G1HtgmGQS_hckuwsa",
   "redirect_uri": "wso2sample://oauth2",
   "authorization_scope": "openid",
-  "discovery_uri": "https://devcloud.kubesb.com/t/usertest.com/oauth2/oidcdiscovery/.well-known/openid-configuration"
+  "discovery_uri": "https://10.0.2.2:9443/oauth2/oidcdiscovery/.well-known/openid-configuration"
```

Comment:
can we use something general like localhost?

Reply:
Resolved in https://github.com/wso2-extensions/identity-sdks-android/pull/15/commits/84a06a42140b0b5d6153ef4b61c9ea3198933898